app/templates/view_times.html

```diff
@@ -59,14 +59,14 @@
     {% endif %}
 
     <div id="paginator"></div>
-
     <div id="view-times">
         <table>
             <thead>
                 <tr>
                     <th></th>
                     <th></th>
                     <th class="alpha">User</th>
+                    <th class="meta">Meta</th>
```

Comment:
This should be changed to `class="alpha"` so times can be sorted by metadata in alphabetical order